pkg/irmacontract.go

```diff
@@ -133,3 +168,42 @@ func validateActingParty(params map[string]string, actingParty string) (bool, er
 	// perform the actual check
 	return actingParty == actingPartyFromContract, nil
 }
+
+func (cv *IrmaContractVerifier) verifyRequiredAttributes(signedIrmaContract *SignedIrmaContract, validationResult *ContractValidationResult) (*ContractValidationResult, error) {
+	if validationResult.ValidationResult == Invalid {
```

Comment:
valdiationResult.ValidationResult naming feels somewhat off? Maybe name the field ContractValidationResult.ValidationResult just Result?

Reply:
Yeah, the naming of this engine is already a disaster. As you can see in this PR I tried to clear up a few things. The fields is exposed via the api like this, so I tend to keep it this name.

Reply:
:+1: